auto_tag/README.md

```diff
@@ -0,0 +1,33 @@
+# Template for adding a GitHub tag from an Azure Pipeline 
+
+Tags are references that point to specific points in Git history. Tagging is generally used to capture a point in history that is used for a marked version release (i.e. v1.0.1).
+The tag name and commit Id on which the tag needs to attached are sent from the calling pipeline as parameters to this template. The tag name can be any string with no spaces and no special characters such as \, ?, ~, ^, :, * , [, @, .. 
+
+## Usage
+
+Reference this repository inside the yaml pipeline
+
+```yaml
+resources:
+  repositories: 
+    - repository: UKHOTemplates
+      type: github
+      endpoint: GitHub_Service_Connection_Name
+      name: UKHO/devops-pipelinetemplates
+```
+
+Steps to add tagging template to your pipeline as job
+
+**Note** : This job should be added at sensible stage in the pipeline, ideally, at the end of the live deployment stage where we can easily see in GitHub what the last commit that got released to production.
```

Comment:
```suggestion
**Note**: This job should be added at a sensible stage in the pipeline to avoid the proliferation of tags. For example, the initial use case in Calypso is after the live deployment stage so we can easily see in GitHub the last commit that was released to production.
```